Documentation/Makefile

```diff
@@ -56,14 +56,19 @@ update-cmdref: builder-image cilium-build ## Update the command reference docume
 	-$(QUIET)rm -rf cmdref/cilium*.md
 	$(QUIET)$(DOCKER_RUN) ./update-cmdref.sh
 
+.PHONY: update-crdlist
+update-crdlist:
+	@$(ECHO_GEN)crdlist
+	make -C ../ generate-crd-docs
+
 codeowners.rst: $(ROOT_DIR)/CODEOWNERS
 	@$(ECHO_GEN)$@
 	$(QUIET)$(DOCKER_RUN) ./update-codeowners.sh
 
 .PHONY: update-codeowners
 update-codeowners: codeowners.rst
 
-check: builder-image api-flaggen update-cmdref update-helm-values update-codeowners ## Validate command and Helm references, as well as policy examples.
+check: builder-image api-flaggen update-cmdref update-crdlist update-helm-values update-codeowners ## Validate command and Helm references, as well as policy examples.
```

Comment:
Here we're adding the update as a dependency for target `check`, but we would also need to add a test below to verify in CI that the update was performed. Would you mind adding something similar to the other `check-*` in the recipe below, please?